Dockerfile.gitlab

```diff
@@ -1,4 +1,4 @@
-FROM registry.ddbuild.io/images/mirror/node:16.16.0-bullseye-slim
+FROM registry.ddbuild.io/images/mirror/ubuntu:24.04
```

Comment:
we probably can use GBI image, which is at registry.ddbuild.io/images/docker:24.0.4-gbi-focal

Reply:
What's the difference with ubuntu ?

Reply:
It is still ubuntu, but `20.04` release labeled as Golden Base Image (GBI), meaning it passed security checks.